test/test_encryption.py

```diff
@@ -1232,16 +1233,13 @@ def test_06_insert_fails_over_16MiB(self):
         doc = {"_id": "encryption_exceeds_16mib", "unencrypted": "a" * (_16_MiB - 2000)}
         doc.update(limits_doc)
 
-        with self.assertRaisesRegex(WriteError, "object to insert too large"):
+        err_msg = "the connected server supports BSON document sizes up to 16777216 bytes."
+        with self.assertRaisesRegex(DocumentTooLarge, expected_regex=err_msg):
             self.coll_encrypted.insert_one(doc)
 
         # Same with bulk_write.
-        doc["_id"] = "encryption_exceeds_16mib_bulk"
-        with self.assertRaises(BulkWriteError) as ctx:
+        with self.assertRaisesRegex(DocumentTooLarge, expected_regex=err_msg):
```

Comment:
Hmm this is surprising. Why does encrypted bulk_write raise DocumentTooLarge now but not regular bulk_write?

Reply:
That doesn’t answer my question though.

Reply:
Where is the test for the regular bulk_write?

Reply:
Or somewhere in `test_bulk.py`?